src/view/standard_fps_movement.cpp

```diff
@@ -0,0 +1,179 @@
+#include "view/standard_fps_movement.hpp"
+#include <SDL.h>
```

Comment:
As I mentioned before, not sure why it isn't "#include <SDL2/SDL.h>"

Reply:
Done